docs/applications_general.py

```diff
@@ -0,0 +1,211 @@
+#In this example, we use a Python data flow scripting interface to connect
+#PyEmma, a simulator adapter, and some additional Python code to iteratively
+#run and analyze groups of simulations until convergence.
+#
+#The main script body defines and executes the subgraph.
+#
+#One supporting module (`analyze.py`) wraps PyEmma.
+#
+#supporting modules (`gromacs_wrapper.py` and `lammps_wrapper.py`) 
+#wraps command line MD commands with a SCALE-MS wrapper layer.  The results should be in the same format.
+
+from gromacs_wrapper import collect_configurations, structure_to_pdb, make_input, simulate, modify_input
+
+# configuration_list (an array of filenames) and md_inputs (a file
+# name, or list of file names if multiple inputs are needed to
+# intitialize the simulation.
+#
+# In theory, this could be abstracted further as some sort of input bundle.
+# that contains all information needed to star the simulation.
+# 
+# So if one has N simulations, you would start with a list of N input objects.
+# Each object could be tagged as a package of a supported type (GROMACS, LAMMPS, NAMD),
+# and wrapper code for each supported type knows how to unwrap it.
+#
+# for GROMACS, it would be a .tpr file (or internal represntation of
+# the .tpr file) For LAMMPS, it's the array of input files, since
+# LAMMPS directly reads that.
+
+coordinate_inputs = configuration_input_list # some list of files
+run_parameters = md_inputs # some list of files
+topology_files = topology_input_list # some list of files
+
+initial_simulation_input = make_input(
+    simulation_parameters=run_parameters,
+    topology=topology_file,
+    conformation=coordinate_inputs)
+
+# Set up an array of N simulations, starting from a single input.
+# MRS: seems like some instructions would need to be given here, such as:
+#       # which entries in the files to replaced by random numbers
+#       # may need a temporary directory to store modified files.
+#       # so maybe the broadcasating would be done in make_input
+
+initial_input = scalems.broadcast(initial_simulation_input, shape=(N,))
+
+# We will need a pdb for MSM building in PyEmma
+initial_pdb = coordinate_inputs[0]
+
+# Get a placeholder object that can serve as a sub context / work graph owner
+# and can be used in a control operation.
+
+# MRS: I'm not sold on using the name subgraph, because it doesn't
+#      describe what it is a subgraph of. Can this be made more specific?
```

Comment:
We document that work flows are represented by directed acyclic graphs, so `subgraph` is the most literal description of what the command is creating. A more generic term could be "fused operation." The symbol to which the subgraph is assigned could more informatively be named `simulation_and_analysis_iteration` or something.

Reply:
> A more generic term could be "fused operation."

I was sort of thinking of a LESS generic term, to be more specific. But naming the output symbol help interpretability a lot.